### ISSUE TITLE
WorldGuard 6 & Thermos B48

### ISSUE THREAD

**Issue by DraksterAU** [TCPR/Thermos#243]
WorldGuard cannot process block break events with Thermos b48.

I am seeing the below message whenever I try to break a block in a region I have permission to (but not admin).
For Ops, blocks break without dropping an entity to pickup.

/wg testbreak -ts {player} results in:

`No plugins cancelled the event. Other causes for cancellation: (1) Bukkit may be using a different event for the action  (example: buckets have their own bucket events); or (2) Minecraft's spawn protection has not been disabled.`

To reproduce: 
Install WorldGuard 6.1 and 6.0 and set up a region only a certain player can build in.
This applies to modded and unmodded blocks.

Did briefly get something that looked like a possible error here using /wg debug testbreak -ts {player} 
http://paste.enginehub.org/VbRhj.txt 
Might be related to what I am targeting... sometimes it comes up with residual heat from railcraft in the output from testbreak.

See the regions.yml file [here](http://stargatemc.com/regions.yml).

**Comment by DraksterAU** [comment]
Oddly enough, players still get the XP drop ... but not the item drop when the block breaks...

**Comment by kristi71111** [comment]
Having this issue aswell.

**Comment by Time6628** [comment]
I believe this was to fix https://github.com/TCPR/Thermos/issues/226

**Comment by DraksterAU** [comment]
Well... That has completely broken world guard then.

**Comment by Time6628** [comment]
Can you post the error without PWP?

**Comment by Robotia** [comment]
This could be caused by the fix for the sugar cane dupe glitch...but I don't see why that would happen.

Could you try both B44 and B45 and let me know if there is any difference?

**Comment by DraksterAU** [comment]
Will do @robotia.  It occurs in KCauldron 185.... Not sure if that means it wasn't caused by the fix

**Comment by Robotia** [comment]
If it occurs in KCauldron then it is not caused by a dupe glitch fix. Could someone please share a video of what the exact problem is?

**Comment by DraksterAU** [comment]
Hi @Robotia 
Sorry I don't have any capture software installed..

Specifically the problem is:

1. Create a global region in WorldGuard and disallow passthrough so that no one can build
2. Create a private region area in WorldGuard and allow members of the region to build
3. Add player X to the private region member list
4. Player X will be able to place blocks in that region, but if he/she breaks them they do not drop entities for pickup

**Comment by Robotia** [comment]
And this is also an issue in KCauldron?

**Comment by DraksterAU** [comment]
@Robotia Yes. It is the same as KCauldron 185, but not on a bukkit server.
